Add an article about using AdGuard VPN in problematic countries (#138)
AdguardTeam/KnowledgeBaseVPN: docs/general/vpn-issues.md

---
title: 'How to use AdGuard VPN in countries with Internet restrictions'
sidebar_position: 10
---

In certain regions, such as China, Iran, and Russia, users might have problems connecting to AdGuard VPN. Here are the options that might help.

> 

Reviewer: First of all, you should explain how to install apps.

iOS: change store in App Store
Android: download apk from the website (mirror website)
Windows: download from the website
macOS: download from the website

## Accessing AdGuard VPN’s website

> If adguard-vpn.com is not accessible, use the mirror websites: [adguard-vpn.info](https://adguard-vpn.info/welcome.html) or [adguardvpn-help.com](https://adguardvpn-help.com/welcome.html).

Author: @ameshkov вот эти сайты указывать?

Reviewer: только один

Reviewer: ```
If adguard-vpn.com is not accessible, try using a [mirror website](https://adguardvpn-help.com/).
```

---
title: 'How to use AdGuard VPN in countries with Internet restrictions'
sidebar_position: 10
---

In certain regions, such as China, Iran, and Russia, users might have problems connecting to AdGuard VPN. Here are the options that might help.

## Accessing AdGuard VPN’s website

If adguard-vpn.com is not accessible, try using the [mirror website](https://adguardvpn-help.com/).

:::info

To avoid potential phishing or malicious websites, only follow mirror links provided in this Knowledge base or by the AdGuard VPN support team.

:::

## Downloading the app

### AdGuard VPN for iOS

> If don't see AdGuard VPN in your App Store, try changing your Apple ID country:

Reviewer 2: ```suggestion
If don't see AdGuard VPN in your App Store, try changing the country or region in your Apple ID account settings:
```

Author: мне кажется account лишнее тут

1. Open the App Store and tap your profile picture.
1. Tap your name, then tap *Country/Region*.
1. Tap *Change Country or Region*.
1. Select a country, then accept the terms and conditions.

[Alternative ways](https://support.apple.com/en-us/HT201389)

> After changing the region, [install AdGuard VPN](https://apps.apple.com/us/app/adguard-vpn-unlimited-fast/id1525373602).

Reviewer 2: ```suggestion
After changing the region, [download AdGuard VPN](https://apps.apple.com/us/app/adguard-vpn-unlimited-fast/id1525373602).
```

Author: из стора же install говорим

### AdGuard VPN for Android

If you have problems with Google Play, download the APK file from [AdGuard VPN's mirror website](https://adguardvpn-help.com/android/overview.html).

### AdGuard VPN for Mac

Download the installation file from [AdGuard VPN's mirror website](https://adguardvpn-help.com/windows/overview.html).

### AdGuard VPN for Windows

Download the installation file from [AdGuard VPN's mirror website](https://adguardvpn-help.com/mac/overview.html).

## Other useful tips

- Always make sure that you are using the latest version of AdGuard VPN, as updates typically include vital fixes and enhancements.

> - Use AdGuard VPN’s full apps instead of browser extensions for a more robust and stable connection.

Reviewer 2: ```suggestion
- Use AdGuard VPN’s full-fledged apps instead of browser extensions for a more robust and stable connection.
```

> - If you encounter problems connecting to a particular location, try connecting to an alternative one. If all locations fail to respond, restart the app to refresh the server list.

Reviewer 2: ```suggestion
- If you encounter problems connecting to a particular location, try connecting to another one. If all locations fail to respond, restart the app to refresh the server list.
```
